spec/lib/stellar/client_spec.rb

```diff
@@ -136,6 +137,37 @@
         end
         expect(native_asset_balance_info["balance"].to_f).to eq 250.0
       end
+
+      it "sends a native payment to the account through a channel account", vcr: {record: :once, match_requests_on: [:method]} do
+        client.create_account(
+          funder: source,
+          account: destination,
+          starting_balance: 100,
+        )
+
+        client.create_account(
+          funder: source,
+          account: channel_account,
+          starting_balance: 100,
+        )
+
+        amount = Stellar::Amount.new(150)
+
+        client.send_payment(
+          from: channel_account,
+          to: destination,
+          amount: amount,
+          source_account: source
+        )
+
+        destination_info = client.account_info(destination)
+        balances = destination_info.balances
+        expect(balances).to_not be_empty
+        native_asset_balance_info = balances.find do |b|
+          b["asset_type"] == "native"
+        end
+        expect(native_asset_balance_info["balance"].to_f).to eq 250.0
+      end
```

Comment:
You should also test that the source account was the one used. This tests that it was sent out, but I think this test would have also passed if you comment out `source_account: source` above.